packages/contracts/contracts/Interfaces/IFeeSharingCollector.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.6.11;
  * @title Interface for contract governance/FeeSharingProxy.sol
```

Comment:
```suggestion  
* @title Interface for Sovryn protocol fee sharing collector  
```

Reply:
Updated in this [commit](https://github.com/DistributedCollective/zero/pull/174/commits/00688cd15e330f453d4ffa48d0f77477b97bcae6)